libs/net/dal/Migrations/1.0.42/Up/PostUp/00-SystemMessage.sql

```diff
@@ -0,0 +1,17 @@
+DO $$
+BEGIN
+
+IF EXISTS (SELECT 1 FROM information_schema.tables WHERE table_name = 'alert') THEN
```

Comment:
Use the EF migration strategy.  This can cause issues.  You can use the PreUp scripts to copy data to a temporary table. Then use the PostUp scripts to update the new table.

You also need to handle rollback.

Reply:
Updated, thanks